ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/transaction/TransactionBackpressureIT.scala

```diff
@@ -56,7 +56,8 @@ class TransactionBackpressureIT
 
       def sendCommands() =
         Source(1 to noOfCommands)
-          .mapAsync(20)(i =>
+          .throttle(10, 1.second)
```

Comment:
Why does not now need a throttle?

Reply:
we need it because otherwise back-pressure kicks in when submitting (SQL is slower than in-memory). I either do exponential backoff of throttle the stream to not overload. Latter was easier here as it's a test.